drivers/clock_control/clock_control_rcar_cpg_mssr.c

```diff
@@ -1,52 +1,22 @@
 /*
- * Copyright (c) 2020 IoT.bzh
+ * Copyright (c) 2020-2022 IoT.bzh
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
 #define DT_DRV_COMPAT renesas_rcar_cpg_mssr
```

Comment:
please change this file to export common function that will be used by each renesas soc specific driver.